### PR TITLE
simplified bump script

### PR DESCRIPTION
**Pull request by redreceipt**

### CODE REVIEW

**Review by vinnyjth — APPROVED** [review]

**Review by vinnyjth — CHANGES_REQUESTED** [review]
image

**Comment by vinnyjth** [comment]
Looks like the output is slightly different. We might need to strip that "v" off the front

**Comment by redreceipt** [comment]
shoot you're right

File: scripts/bump.sh [hunk 1]
```diff
@@ -7,14 +7,8 @@ curl 'https://api.github.com/repos/apollosproject/apollos-apps/tags'
...
-	curl -s 'https://api.github.com/repos/apollosproject/apollos-apps/tags' |
-		python -m json.tool |
-		grep '\"name\"' |
-		cut -d ':' -f 2 |
-		sed 's/&quot;/\"/g' |
-		sed -E 's/\"v(.*)\".*/\1/g' |
-		sed -n 1p |
-		tr -d '[:space:]'
+	curl -s "https://api.github.com/repos/apollosproject/apollos-apps/tags" |
+		python -c "import sys, json; print json.load(sys.stdin)[0]['name']"
```
Comment (redreceipt) [inline scripts/bump.sh:11]:
```suggestion
		python -c "import sys, json; print json.load(sys.stdin)[0]['name'][1:]"
```

**Comment by redreceipt** [comment]
actually now I'm worried about the carriage return 😂 just a sec

**Comment by redreceipt** [comment]
ok i think this will work

**Comment by vinnyjth** [comment]
@redreceipt yah I ran some tests locally and things looked fine - the `npm` command also has a carriage return.

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**